oonib/report/handlers.py

```diff
@@ -126,63 +138,58 @@ def parseNewReportRequest(request):
         'software_version': version_string,
         'test_name': name,
         'test_version': version_string,
-        'probe_asn': probe_asn
+        'probe_asn': probe_asn,
+        'probe_cc': probe_cc,
+        'data_format_version': version_string
     }
 
-    parsed_request = json.loads(request)
-    if 'probe_asn' not in parsed_request or not parsed_request['probe_asn']:
-        parsed_request['probe_asn'] = 'AS0'
+    if not header.get('probe_asn'):
+        header['probe_asn'] = 'AS0'
+
+    if not header.get('probe_cc'):
+        header['probe_cc'] = 'ZZ'
+
+    if not header.get('start_time'):
+        header['start_time'] = time.time()
+
+    if not header.get('start_time'):
+        header['start_time'] = time.time()
```

Comment:
yeah you are right. I suspect this may be due to some rebase mess up. Anyways very good catch.